config/locales/en/errors.yml

```diff
@@ -148,6 +148,24 @@ en:
               invalid_year: Enter a valid year
               year_too_early: Date your client last attended an appointment with their JSA work coach is too far in the past
               future_not_allowed: Date your client last attended an appointment with their JSA work coach must be today or in the past
+        steps/dwp/partner_benefit_type_form:
```

Comment:
This will probably require a helper but I'm deferring refactoring in this pr as I think the solution for how best to refactor this to avoid duplication will become clearer when we start implementing the shared partner client forms.